mesh/nurbs.cpp

```diff
@@ -541,7 +541,7 @@ void KnotVector::FindMaxima(Array<int> &ks, Vector &xi, Vector &u) const
             CalcShape(shape, i, arg1);
             max1 = shape[d];
 
-            arg2 = 1-(1e-16);
+            arg2 = 1 - (1e-16);
```

Comment:
Typically, the largest `real_t` number smaller than 1 is given by `1_r - eps/2_r` (or simply `1 - eps/2`) where `eps = std::numeric_limits<real_t>::epsilon()`.

Reply:
Done in 2d3aba5d8727f9776c28b688d046b890585726dc